core/src/main/java/com/group5/core/physics/PlayerPhysicsStrategy.java

```diff
@@ -84,7 +86,8 @@ public void jump(final float intensity) {
      */
     public void kill() {
         Body b = getBody();
-        World.setVelocityThreshold(100000000);
+        b.setFixedRotation(false);
+        World.setVelocityThreshold(Integer.MAX_VALUE);
```

Comment:
Nice that you replaced the magic number with this, more descriptive of it's use this way.